lib/fast_jsonapi/serialization_core.rb

```diff
@@ -149,6 +149,7 @@ def get_included_records(record, includes_list, known_included_objects, params =
                 included_records.concat(serializer_records) unless serializer_records.empty?
               end
 
+              next unless inc_obj
```

Comment:
I ran into the same error. Seems like they're trying to solve the issue above on line 144:
```
included_objects = [included_objects] unless relationship_type == :has_many
next if included_objects.blank?
```

The problem is that included objects is an array containing an emtpy element `[nil]`

I think if we just switch the order of the above 2 methods it will solve the problem and then you dont need to add this nil check

```
next if included_objects.blank?
included_objects = [included_objects] unless relationship_type == :has_many
```